### PR TITLE
VM tutorial | Added equivalent PowerShell command code blocks

### PR DESCRIPTION
**Pull request by v-abdriaz**
Added PowerShell commands for creating shell variables and creating a virtual network and subnet.

### CODE REVIEW

**Comment by v-abdriaz** [comment]
#Sign-off

**Comment by dbradish-microsoft** [comment]
@v-abdriaz, all new PowerShell scripts have been tested in PowerShell via Azure Cloud Shell, or PowerShell via PowerShell v7?

File: docs-ref-conceptual/azure-cli-vm-tutorial-5.md [hunk 1]
```diff
@@ -22,13 +22,25 @@ az network nic show --ids $nicId
 
 This command shows all of the information for the network interface of the VM. This data includes DNS settings, IP information, security settings, and the MAC address. The following query shows how to obtain the public IP address and subnet object IDs.
 
+# [Bash](#tab/bash)
+
 ```azurecli-interactive
 az network nic show --ids $nicId \
-  --query '{IP:ipConfigurations[].publicIpAddress.id, Subnet:ipConfigurations[].subnet.id}' \
+  --query '{IP:ipConfigurations[].publicIPAddress.id, Subnet:ipConfigurations[].subnet.id}' \
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-5.md:29]: @v-abdriaz , This command returned no results b/c JMESPath is case sensitive.  `publicIpAddress` is not the same as `publicIPAddress`.  If I recall correctly, this spelling was changed after this article was originally written, but having an empty `$nicId` makes the following steps not work correctly.

In the following screen print, there was no error, but the command didn't work.  When you test a code block, please verify both **errors and results**.

image

I have fixed this spelling in your branch so I can merge the PR.

File: docs-ref-conceptual/azure-cli-vm-tutorial-4.md [hunk 2]
```diff
@@ -28,13 +28,26 @@ to get more information about the VM's configuration and capabilities.
...
+# [Bash](#tab/bash)
+
 ```azurecli-interactive
 az vm show --name $vmName \
   --resource-group $resourceGroup \
   --query 'networkProfile.networkInterfaces[].id' \
   --output tsv
 ```
 
+# [PowerShell](#tab/powershell)
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-4.md:40]: @v-abdriaz , to be consistent with the rest of the article, I added both a Bash and PowerShell conceptual tab here. The only difference is line continuation characters, but we've made this distinction in the other steps, so we need to do the extra work here as well.

This update was completed in your branch so I can merge the PR.

File: docs-ref-conceptual/azure-cli-vm-tutorial-5.md [hunk 3]
```diff
@@ -45,21 +57,54 @@ for command-line tools, it helps with human readability and can be used with cus
...
+  --query '[ipConfigurations[].publicIPAddress.id, ipConfigurations[].subnet.id]' \
   -o tsv)
 ```
 
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+$output = az network nic show --ids $nicId --query "[ipConfigurations[].publicIPAddress.id, ipConfigurations[].subnet.id]" -o tsv
+
+# Split the output into two variables
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-5.md:74]: @v-abdriaz , nice work here!

File: docs-ref-conceptual/azure-cli-vm-tutorial-1.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Create virtual machine (VM) on a virtual network (VNet) prerequisites – Azure CLI | Microsoft Docs
 description: Prerequisites for creating a virtual machines (VM) on a virtual network (VNet) with the Azure CLI.
-ms.date: 08/2/2023
+ms.date: 01/08/2024
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-1.md:4]: @v-abdriaz , Please remember to update the `ms.date` when you refresh an article.  I have completed this in all 5 files within your branch.  Just remember for next time.

File: docs-ref-conceptual/azure-cli-vm-tutorial-3.md [hunk 5]
```diff
@@ -38,14 +41,33 @@ az vm create \
...
+
 > [!NOTE]
 > If you have an SSH key named `id_rsa` already available, this key is used for authentication rather than having a new
 > key generated.
 
 As the VM is created, you see the local values used and Azure resources being created due to the `--verbose` option.
 Once the VM is ready, a JSON is returned from the Azure service including the public IP address.
 
-```json
+```output
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-3.md:70]: @v-abdriaz , output code blocks should always have an `output` code tag regardless of table output, json, etc.  See Contributor's Guide [Output code blocks](https://review.learn.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=main#output).

File: docs-ref-conceptual/azure-cli-vm-tutorial-5.md [hunk 6]
```diff
@@ -45,21 +57,54 @@ for command-line tools, it helps with human readability and can be used with cus
...
+$ipId, $subnetId = $output -split "`t"
+```
+
+---
+
+If you don't want to use the Bash `read` or PowerShell `-split` command, you can set each variable separately.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-5.md:80]: @v-abdriaz , not everyone wants to mix Bash commands and Azure CLI, but I just stuck this in here as I used it for testing.  This isn't something I'd expect you to add.

**Pull request merged**